src/smooch.js

```diff
@@ -7,7 +7,8 @@ import packageInfo from '../package.json';
 export const SERVICE_URL = 'https://api.smooch.io/v1';
 
 export class Smooch {
-    constructor(auth = {}, serviceUrl = SERVICE_URL) {
+    constructor(auth = {}, options = {}) {
+        const {serviceUrl = SERVICE_URL, headers = {}} = options;
```

Comment:
you could do that right in the param definition `{serviceUrl, headers} = {serviceUrl = SERVICE_URL, headers = {}}` but okay